src/test/ui/derives/derives-span-PartialEq-tuple-struct.stderr

```diff
@@ -3,6 +3,9 @@ error[E0369]: binary operation `==` cannot be applied to type `Error`
    |
 LL |     Error
    |     ^^^^^
+   |     |
+   |     Error
+   |     Error
```

Comment:
Can you add a check for the case when both left and right overlap to avoid this output?

Reply:
This is fixed